test/issue-generator.js

```diff
@@ -64,4 +91,22 @@ export default function createTestIssue(key, type, color, parentKey, parentType,
     case 'Task':
       return baseIssue;
   }
+
+}
+
+export function createIssueWithMutipleLinks(key, type, color, parents)
+{
+  let baseIssue = createBaseIssue(key, type, color);
+  _.forEach(parents, function(n)
+  {
+    baseIssue = resolveIssue(baseIssue, type, n.key, n.type, n.linkType);
+  });
+  return baseIssue;
+}
+
+export {createIssueWithMutipleLinks as createIssueWithMutipleLinks};
```

Comment:
This `export {createIssueWithMutipleLinks as createIssueWithMutipleLinks}` is unnecessary.